docs/content/concepts/basic-concepts.md

```diff
@@ -40,7 +40,7 @@ By partitioning, users can efficiently operate on a slice of records in the tabl
 
 {{< hint info >}}
 
-Partition keys must be a subset of primary keys if primary keys are defined.
+Partition keys must be a subset of primary keys if primary keys are defined and the table is using fixed bucket mode. For tables using dynamic bucket mode, partition keys only need to be different from the primary keys, see [Dynamic Bucket]({{< ref "concepts/primary-key-table#dynamic-bucket">}}).
```

Comment:
`If you want to use cross partition update, you should use dynamic bucket mode.`?

Reply:
> `If you want to use cross partition update, you should use dynamic bucket mode.`?

@JingsongLi Hi, pls check the updated version.